feedback-module/.gitignore

```diff
@@ -22,3 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
```

Comment:
what about the .DS store files?

Reply:
The .DS_Store was already in the .gitignore— I think at some point in the beginning we might've accidentally pushed everything to dev without the .gitignore and then forgot to remove it?